src/Group.php

```diff
@@ -83,5 +121,18 @@ public function getMembers(): array
     {
         return $this->members;
     }
-
+    /**
+     * @param User $user
+     * @return void
+     */
+    public function addUserToGroup($user)
```

Comment:
If possible add the types into the function declaration
```suggestion
    public function addUserToGroup(User $user): void
```